.github/workflows/android-app.yml

```
@@ -6,11 +6,12 @@ on:
     branches:
       - master
       - stable
+      - change-android-runner
 
 jobs:
   build:
-    name: Build
-    runs-on:            ubuntu-latest
+    name: Build android
```

```suggestion
    name: Build
```

As `Build` is required step in repo settings and we shouldn't change it

---

Fixed, 
but I see this is the only flow and job called Build. Do you know which requirement it is?